docs/guide/vue-webmcp-best-practice.md

```
@@ -0,0 +1,549 @@
+| **WebMCP Server** | `@opentiny/next-sdk` | 在浏览器中运行的 MCP 工具服务器，注册可供 AI 调用的工具 |
+| **Page Tool Bridge** | `@opentiny/next-sdk` | 工具调用时自动导航到目标页面，并通过消息通信执行页面内逻辑 |
+| **WebSkills** | `@opentiny/next-sdk` + `@opentiny/next-remoter` | 结构化知识包，让 AI 获得特定领域的角色和文档知识 |
+| **TinyRemoter** | `@opentiny/next-remoter` | AI 对话面板组件，集成 LLM + MCP + Skills |
+
+### 为什么需要 Page Tool Bridge？
+
+Web MCP 与传统 MCP（运行在服务器/进程中）的本质区别在于：**Web MCP 工具是动态的、随页面生命周期开启和关闭的**。
+
+用户不一定打开了工具对应的页面，Page Tool Bridge 解决了这个问题：
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)